### ISSUE TITLE
Possibility of adding more than one mesh to the plot

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/qgis-crayfish-plugin#393]

**Comment by PeterPetrik** [comment]
@saberraz can you please elaborate on description and use-case? I do not understand the task

**Comment by saberraz** [comment]
Currently only one quantity (e.g. water depth) can be added to a plot. Would be good to have an option to add multiple quantities (even from different meshes) to the same plot.

**Comment by mirko-sessa** [comment]
Such a new feature would be really useful.
I work for the dam office of my region and this feature would be beneficial, for instance, to compare on the same chart:

- the total precipitation coming from the GRIB forecast, and
- the total precipitation measured by our rain-gauge network.

**Comment by ifariasg** [comment]
Indeed, this feature is something that can be really useful when comparing datasets. 
Right now the only work around is to substract datasets to achieve a similar results. Or get a set of points\lines, evaluate them in the different datasets, post process to finally judge the results.

I work with hydrological models that save all the results to netcdf. Although visualization and data export can be handled easily with crayfish, on the fly comparisons are not.
